### PR TITLE
 Translate Ruby 2.6.0 Release (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Ref: https://github.com/ruby/www.ruby-lang.org/issues/1894

This translates https://github.com/ruby/www.ruby-lang.org/pull/1926

Check 0dd4036 for review, @ruby/www-ruby-lang-org-i18n-ko :pray:

### CODE REVIEW

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+  `Proc#call`을 대량으로 호출하는 `lc_fizzbuzz` 벤치마크가 1.4배 빨라졌습니다. [[Bug #10212]](https://bugs.ruby-lang.org/issues/10212)
+
+* `block`이 블록 파라미터인 경우의 `block.call`이 빨라졌습니다. [[Feature #14330]](https://bugs.ruby-lang.org/issues/14330)
+
+  루비 2.5에서 개선된 블록 넘기기 기능을 포함하여, 루비 2.6에서는 블록 평가가 간단한 벤치마크에서 2.6배 빨라졌습니다. [[Feature #14045]](https://bugs.ruby-lang.org/issues/14045),
+
+* Transient Heap(theap)이 도입되었습니다. [[Bug #14858]](https://bugs.ruby-lang.org/issues/14858) [[Feature #14989]](https://bugs.ruby-lang.org/issues/14989)
+
+  `theap`은 특정 클래스(`Array`, `Hash`, `Object`, `Struct`)가 가리키는 짧은 생애를 가지는 메모리 객체들을 관리합니다. 예를 들어 작고 짧게 생존하는 Hash 객체는 2배 빨라집니다. rdoc 벤치마크에서 6-7%의 성능 향상을 확인했습니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]: Can you also apply this to preview3, rc1, and rc2 news?
```suggestion
  `theap`은 특정 클래스(`Array`, `Hash`, `Object`, `Struct`)가 가리키는 짧은 생애를 가지는 메모리 객체들을 관리합니다. 예를 들어 작고 짧게 생존하는 Hash 객체의 생성이 2배 빨라집니다. rdoc 벤치마크에서 6-7%의 성능 향상을 확인했습니다.
```

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+  `Fiber.yield`와 `Fiber#resume`은 64비트 리눅스에서 5배 빨라집니다. Fiber를 빈번하게 사용하는 프로그램에서는 전체적으로 5%의 성능 향상을 기대할 수 있습니다.
+
+## 2.5 이후 주목할 만한 변경
+
+* `$SAFE`가 프로세스 전역 변수로 취급되며, `0` 이외의 값을 설정한 후에 `0`으로 되돌리는 것이 가능해집니다. [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
+
+* `ERB.new`에 `safe_level`을 넘기는 기능이 제거될 예정입니다. 또한 `trim_mode`와 `eoutvar`는 키워드 변수로 변경됩니다. [[Feature #14256]](https://bugs.ruby-lang.org/issues/14256)
+
+* 유니코드 지원 버전이 11으로 갱신되었습니다. 이는 루비 2.6의 TEENY 릴리스에서 12와 12.1로 갱신될 예정입니다. 이는 [새 일본 연호](http://blog.unicode.org/2018/09/new-japanese-era.html) (영문)에 대한 지원이 포함됩니다.
```
Comment (yous) [inline ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]:
```suggestion
* 유니코드 지원 버전이 11로 갱신되었습니다. 이는 루비 2.6의 TEENY 릴리스에서 12와 12.1로 갱신될 예정입니다. 이는 [새 일본 연호](http://blog.unicode.org/2018/09/new-japanese-era.html)(영문)에 대한 지원을 포함합니다.
```

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+* RubyGems 3.0.1을 병합했습니다. `--ri`와 `--rdoc` 옵션이 제거되었습니다. 대신에 `--document`와 `--no-document`를 사용해주세요.
+
+* [Bundler](https://github.com/bundler/bundler)를 기본 젬으로 병합했습니다.
+
+* `rescue`가 없는 `else`가 문법 에러가 됩니다. [EXPERIMENTAL] [[Feature #14606]](https://bugs.ruby-lang.org/issues/14606)
+
+[NEWS](https://github.com/ruby/ruby/blob/v2_6_0/NEWS)나 [커밋 로그](https://github.com/ruby/ruby/compare/v2_5_0...v2_6_0)에서 더 자세한 설명을 확인할 수 있습니다.
+
+이러한 변경 사항에 따라, 루비 2.5.0 이후로 [파일 6437개 수정, 231471줄 추가(+), 98498줄 삭제(-)](https://github.com/ruby/ruby/compare/v2_5_0...v2_6_0_rc2)가 이루어졌습니다!
```
Comment (yous) [inline ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]:
```suggestion
이러한 변경 사항에 따라, 루비 2.5.0 이후로 [파일 6437개 수정, 231471줄 추가(+), 98498줄 삭제(-)](https://github.com/ruby/ruby/compare/v2_5_0...v2_6_0)가 이루어졌습니다!
```

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+메리 크리스마스, 행복한 휴일 보내시고, 루비 2.6와 함께 즐거운 프로그래밍 하세요!
```
Comment (yous) [inline ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]:
```suggestion
메리 크리스마스, 행복한 휴일 보내시고, 루비 2.6과 함께 즐거운 프로그래밍 하세요!
```

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]: 🙈

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]: 82382542 👍

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+* ASCII 이외의 대문자로 시작하는 상수를 정의할 수 있게 됩니다. [[Feature #13770]](https://bugs.ruby-lang.org/issues/13770)
+
+* 종료 지정이 없는 범위 연산자가 추가됩니다. [[Feature #12912]](https://bugs.ruby-lang.org/issues/12912)
+
+  종료 지정이 없는 범위 연산자 `(1..)`가 추가됩니다. 이는 끝이 없는 것처럼 취급됩니다. 다음은 전형적인 사용 예시입니다.
+
+      ary[1..]                             # ary[1..-1]와 동치
+      (1..).each {|index| block }          # 1로 시작하는 무한 루프
+      ary.zip(1..) {|elem, index| block }  # ary.each.with_index(1) { ... }
```
[the remark this answers is not in the document]
Comment (yous) [reply ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]: This one is not yet applied. :smiley:

[the remark this answers is not in the document]
Comment (riseshia) [reply ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]: Oops e933035

**Comment by riseshia** [comment]
Thanks for your review!! :bow:

File: ko/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+* `Kernal#system`이 실패했을 경우 `false`를 돌려주는 대신, 에러를 던지도록 하는 `:exception` 옵션을 추가했습니다. [[Feature #14386]](https://bugs.ruby-lang.org/issues/14386)
+
+* `Coverage`의 oneshot_lines 모드를 추가했습니다. [[Feature#15022]](https://bugs.ruby-lang.org/issues/15022)
+
+  * 이 모드는 '각 줄이 몇 번 실행되었는지' 대신 '각 줄이 한 번 이상 실행되었는지'를 확인합니다. 각 줄의 훅은 최대 1회만 실행되며, 실행된 후에는 플래그를 제거하기 때문에 오버헤드 없이 실행됩니다.
+  * `Coverage.start`에 `:oneshot_lines` 키워드 인수가 추가됩니다.
+  * `Coverage.result`에 `:stop`과 `:clear` 키워드 인수가 추가됩니다. 만약 `clear`가 참이라면, 이는 카운터를 0으로 초기화합니다. 만약 `stop`이 참이라면 커버리지 측정을 비활성화합니다.
+  * 주어진 소스 코드로부터 'stub'을 생성하는 간단한 헬퍼 함수인 `Coverage.line_stub`을 추가합니다.
+
+* `FileUtils#cp_lr`을 추가했습니다. 이는 cp_r 처럼 동작하지만, 복사를 하는 대신 링크를 생성합니다. [[Feature #4189]](https://bugs.ruby-lang.org/issues/4189)
```
Comment (yous) [inline ko/news/_posts/2018-12-25-ruby-2-6-0-released.md]: One more update: https://github.com/ruby/www.ruby-lang.org/commit/43399ee7c0dd1a310d839101623d97e1892fe32b

```suggestion
* `FileUtils#cp_lr`을 추가했습니다. 이는 `cp_r`처럼 동작하지만, 복사를 하는 대신 링크를 생성합니다. [[Feature #4189]](https://bugs.ruby-lang.org/issues/4189)
```

**Pull request merged**